### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by lyndsysimon**

### CODE REVIEW

File: tests/test_explicit_keys.py [hunk 1]
```diff
@@ -23,23 +23,23 @@ class Delta(Charlie):
 
 
 def test_tracked_descendants():
```
Comment (pramttl) [inline tests/test_explicit_keys.py]: (Optional) Could be updated to `test_progeny` to match the property method name. Generally on the pbm codebase we try to keep the test names something like `test_method_name` where `method_name` is the method being tested.

File: tests/test_simple_hierarchy.py [hunk 2]
```diff
@@ -18,29 +18,29 @@ class Delta(Charlie):
 
 
 def test_tracked_descendants():
```
Comment (pramttl) [inline tests/test_simple_hierarchy.py]: Isn't this a duplicate test ? (Also present above in, test_explicit_keys.py)

File: tests/test_tracking_flag.py [hunk 3]
```diff
@@ -22,18 +22,18 @@ class Delta(Charlie):
 
 
 def test_tracked_descendants():
```
Comment (pramttl) [inline tests/test_tracking_flag.py]: Another duplicate ?

File: tests/test_explicit_keys.py (hunk 1 above)
Comment (lyndsysimon) [reply tests/test_explicit_keys.py]: Yeah, this test name should be updated.

File: tests/test_simple_hierarchy.py (hunk 2 above)
Comment (lyndsysimon) [reply tests/test_simple_hierarchy.py]: Not duplicate, no - the setup is different, as are the assertions. They're testing the different results of the same method under different circumstances.

File: src/progeny/__init__.py [hunk 4]
```diff
@@ -1,6 +1,22 @@
 import six
 
 
+class ProgenySet(set):
```
Comment (rouge8) [inline src/progeny/__init__.py]: Implement `collections.Set` maybe? https://docs.python.org/3/library/collections.abc.html or at a minimum, this should probably be a frozenset (and if we do freeze it, test(s) to verify it's frozen 😃)

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+sudo: false
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+install:
+  - pip install tox-travis pycodestyle
+before_script:
```
Comment (rouge8) [inline .travis.yml]: @lyndsysimon I might push out a PR tweaking the Travis and tox setup

File: src/progeny/__init__.py [hunk 6]
```diff
@@ -17,31 +33,25 @@ def __init__(cls, name, bases, dct):
         if '__progeny_key__' not in dct.keys():
             cls.__progeny_key__ = cls._get_progeny_key()
 
+    @property
+    def progeny(self):
```
Comment (rouge8) [inline src/progeny/__init__.py:37]: would progeny classes (e.g. `Alpha`, `Bravo`, etc. in the tests) also have a `.progeny`?

**Review by rouge8 — COMMENTED** [review]
Let's see if I understand this, adapting it to our `Backend` model:

```python
class Backend(ProgenyBase):
    def _get_progeny_key(cls):
        return cls.id

class Backend1(Backend):
    id = 'ONE'

class Backend2(Backend):
    id = 'TWO'
```

@lyndsysimon that look right?

File: .travis.yml (hunk 5 above)
Comment (rouge8) [reply .travis.yml]: https://github.com/GoodRx/progeny/pull/4

File: src/progeny/__init__.py (hunk 6 above)
Comment (rouge8) [reply src/progeny/__init__.py:37]: to answer my own question: they do, which I don't love. I'm not sure if it'd be more confusing if they didn't have it nor how much effort. no strong opinion (and definitely not a blocker), just food for thought

Comment (lyndsysimon) [reply src/progeny/__init__.py:37]: They should, yes. That behavior is not explicitly tested so I'm not going to promise that it works, but that was my intention.

File: README.rst [hunk 7]
```diff
@@ -53,10 +59,10 @@ In some cases, it may be useful to prevent descendant classes from being visible
 
 .. code:: python
 
-    from progeny import ProgenyBase
+    from progeny import progeny.Base
```
Comment (rouge8) [inline README.rst:62]: bad find/replace here

File: README.rst [hunk 8]
```diff
@@ -43,7 +49,7 @@ Now we can iterate over all of the subclasses of ``NotificationHandler``:
 .. code:: python
 
     def send_newsletter():
-        for handler in NotificationHandler.tracked_descendants():
+        for handler in NotificationHandler.progeny:
```
Comment (rouge8) [inline README.rst]: `.progeny.values()`

**Review by rouge8 — CHANGES_REQUESTED** [review]
Two README fixes that could easily have been my fault 🙈

**Review by rouge8 — APPROVED** [review]

**Pull request merged**